docs/configuration-format.md

```diff
@@ -0,0 +1,157 @@
+# OpenTelemetry Collector Configuration Format
+
+The effective configuration that defines runtime behavior of the collector is calculated from 2 sources: the Default configuration which is hard-coded and the User-defined configuration which supplied by the user via various configuration sources. Default configuration defines reasonable defaults that most users will likely want to use. User-defined configuration can override the defaults and specify additional configuration. The effective configuration is calculated by merging Default and User-defined configurations (see merging rules in a separate section below).
+
+The top-level entities in the configuration file are receivers, processors, exporters, pipeline (more about each below in separate sections). Here is the top-level structure of the configuration:
+
+```yaml
+receivers:
+  # map of receivers
+processors:
+  # map of processors
+exporters:
+  # map of exporters
+extensions:
+  # map of extensions
+pipelines:
```

Comment:
We changed that between the doc and the implementation :) we have a `service` section.